src/Chunk.cpp

```diff
@@ -741,6 +767,29 @@ void cChunk::MoveEntityToNewChunk(cEntity * a_Entity)
 
 
 
+void cChunk::UpdateCanUnload()
```

Comment:
i.e. What if someone forgets to call it and introduces another hard to trace bug? I'm of the opinion that its generally better not to store easily calculable state; but, I cannot currently suggest a replacement bar calculating CanUnloadCount manually.